src/edu/csus/ecs/pc2/core/execute/ExecuteTimerFrame.java

```diff
@@ -0,0 +1,237 @@
+        } catch (java.lang.Throwable ivjExc) {
+            handleException(ivjExc);
+        }
+
+        return contentPaneBorderLayout;
+    }
+
+    public void resetFrame()
+    {
+        if(ivjExecuteTimerLabel != null) {
+            ivjExecuteTimerLabel.setText("Execution Time");
+            ivjExecuteTimerLabel
+                    .setVerticalTextPosition(javax.swing.SwingConstants.BOTTOM);
+            ivjExecuteTimerLabel
+                    .setFont(new java.awt.Font("dialog", 1, 18));
+            ivjExecuteTimerLabel
+                    .setVerticalAlignment(javax.swing.SwingConstants.BOTTOM);
+            ivjExecuteTimerLabel
+                    .setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
+            ivjExecuteTimerLabel.setCursor(new java.awt.Cursor(
+                    java.awt.Cursor.DEFAULT_CURSOR));
+        }
+        if(ivjTimerCount != null) {
+            ivjTimerCount.setForeground(java.awt.Color.black);
+            ivjTimerCount.setFont(new java.awt.Font("monospaced", 1, 48));
+            ivjTimerCount.setText(" ");
+            ivjTimerCount
+                    .setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
+            ivjTimerCount.setCursor(new java.awt.Cursor(
+                    java.awt.Cursor.DEFAULT_CURSOR));
+        }
+        if(ivjbtnTerminate != null) {
+            ivjbtnTerminate.setMnemonic('t');
+            ivjbtnTerminate.setText("Terminate");           
+        }
+    }
+    
+    public void setTimerFrameVisible(boolean bVis)
+    {
+        setVisible(bVis);
+    }
+    
+    /**
+     * Return the ExecuteTimerLabel1 property value.
+     *
+     * @return javax.swing.JLabel
+     */
+    private JLabel getExecuteTimerLabel() {
+        if (ivjExecuteTimerLabel == null) {
+            try {
+                ivjExecuteTimerLabel = new javax.swing.JLabel();
+                ivjExecuteTimerLabel.setName("ExecuteTimerLabel");
+                ivjExecuteTimerLabel.setText("Execution Time");
+                ivjExecuteTimerLabel
+                        .setVerticalTextPosition(javax.swing.SwingConstants.BOTTOM);
+                ivjExecuteTimerLabel
+                        .setFont(new java.awt.Font("dialog", 1, 18));
+                ivjExecuteTimerLabel
+                        .setVerticalAlignment(javax.swing.SwingConstants.BOTTOM);
+                ivjExecuteTimerLabel
+                        .setHorizontalAlignment(javax.swing.SwingConstants.CENTER);
+                ivjExecuteTimerLabel.setCursor(new java.awt.Cursor(
+                        java.awt.Cursor.DEFAULT_CURSOR));
+            } catch (java.lang.Throwable ivjExc) {
+                handleException(ivjExc);
+            }
+        }
+        return ivjExecuteTimerLabel;
+    }
+    
+    /**
+     * Return the TimerCount property value.
+     *
+     * @return javax.swing.JLabel
+     */
+    private JLabel getTimerCountLabel() {
+        if (ivjTimerCount == null) {
+            try {
```

Comment:
Same comment as above; the body of this `try` seems to be virtually identical to most of the (second) `if` statement in method `resetFrame()` and should probably be encapsulated into its own method.

Reply:
Agreed, but they are not exactly the same.  One sets the color back to the default, but, yes, the other stuff is the same.

Reply:
My question, as above, is:  do you intend to change this?  (I was hoping that since you "Agreed", it meant you'd pull the common functions out into a single method, call that method, and then have each block perform whatever additional steps are needed to account for the differences..)

Reply:
This has been fixed and pushed.